api/views/assets.py

```diff
@@ -78,6 +78,11 @@ def get_object(self):
         obj = get_object_or_404(queryset, uuid=self.kwargs["pk"])
         return obj
 
+    @staticmethod
+    def restrict_to_only_super_users(invoice_receipt, user):
+        if invoice_receipt and user.is_superuser is not True:
```

Comment:
`user.is_superuser` is a boolean value so you could use it directly here:`if invoice_receipt and not user.is_superuser`